test/tests/ver-1b/comparison_ver-1b.py

```diff
@@ -43,7 +43,7 @@
 gs = gridspec.GridSpec(1,1)
 ax = fig.add_subplot(gs[0])
 
-tmap_sol = pd.read_csv("./gold/u_vs_x.csv")
+tmap_sol = pd.read_csv("ver-1b_vector_postproc_line_0250.csv")
```

Comment:
```suggestion
tmap_sol = pd.read_csv("./gold/ver-1b_vector_postproc_line_0250.csv")
```
We settled on using the gold files.